kod/object/active/holder/nomoveon/battler/monster/towns/barlqtwn/bqinnk.kod

```diff
@@ -30,21 +30,16 @@ resources:
    BarloqueInnkeeper_room_rented = "Here is the key to thine room.  I trust you will findest "
       "our rooms to be most well-appointed.  Enjoyest thy stay!"
 
-   BarloqueInnkeeper_cost1 = "The rooms here are the finest.  I needeth "
-   BarloqueInnkeeper_cost2 = " shillings when thou ~Irent~neth a ~Iroom~n for for 85 days.  After that, I requireth "
-   BarloqueInnkeeper_cost3 = " per day up front.  Key copies costeth "
-   BarloqueInnkeeper_cost4 = " shillings if thou wishest to share thine room with a special someone."
+   BarloqueInnkeeper_cost = \
+      "The rooms here are the finest.  I needeth %i shillings when thou ~Irent~neth a "
+      "~Iroom~n for for 85 days.  After that, I requireth %i per day up front.  Key copies "
+      "costeth %i shillings if thou wishest to share thine room with a special someone."
 
-   BarloqueInnkeeper_days_left = "I thanketh thee.  Thou hast %q days left at my luxurious inn."
+   BarloqueInnkeeper_days_left = "I thanketh thee.  Thou hast %i days left at my luxurious inn."
    BarloqueInnkeeper_too_long = "It pleaseth me that ye liketh thine room, but thou hast paid enough in rent already!  I canst not accept more from you right now."
 
-   BarloqueInnkeeper_destroy_key_copy = "~kPritchett tells you, \"I will discardeth the key for thou.\""
+   BarloqueInnkeeper_destroy_key_copy = "I will discardeth the key for thou."
```

Comment:
This should be "for thee" (I know it was wrong originally).

Reply:
Should be fixed now.